packages/liferay-npm-scripts/src/config/jest.config.js

```diff
@@ -17,7 +17,7 @@ module.exports = {
 	testResultsProcessor: 'liferay-jest-junit-reporter',
 	testURL: 'http://localhost',
 	transform: {
-		'.+': path.join(__dirname, '..', 'jest', 'transformBabel.js'),
-		'\\.soy$': path.join(__dirname, '..', 'jest', 'transformSoy.js')
+		'\\.soy$': path.join(__dirname, '..', 'jest', 'transformSoy.js'),
+		'.+': path.join(__dirname, '..', 'jest', 'transformBabel.js') // eslint-disable-line sort-keys
```

Comment:
FWIW this is probably less obvious and more fragile than:

```javascript
		// eslint-disable sort-keys
		'.+': path.join(__dirname, '..', 'jest', 'transformBabel.js'),			'\\.soy$': path.join(__dirname, '..', 'jest', 'transformSoy.js'),
		'\\.soy$': path.join(__dirname, '..', 'jest', 'transformSoy.js')
		// eslint-enable sort-keys
```

(in the event that the list grows over time). Same below, although I don't think it is going to grow much, if ever.

Reply:
got it! wasn't aware of that syntax... I can push force with it, just to stay on the safe side